docs/src/gs/create-workload-cluster.md

```diff
@@ -52,16 +58,20 @@ clusterctl generate cluster <cluster-name>\
 
 ## Create a new workload cluster on bare metal instances using an Ubuntu custom image
 
-Note the addition of `OCI_PV_TRANSIT_ENCRYPTION=false` which is required for most BM shapes.
+Note the addition of the `OCI_CONTROL_PLANE_SHAPE` variables, `OCI_WORKER_SHAPE` variables to change the shape 
+information from the `VM.Standard.E4.Flex` default. You will also need to set`OCI_PV_TRANSIT_ENCRYPTION=false` which is 
+required for most BM shapes.
```

Comment:
```suggestion
The following command uses the `OCI_CONTROL_PLANE_SHAPE` and `OCI_WORKER_SHAPE` parameters to specify bare metal shapes instead of using CAPOCI's default virtual instance shape. The `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` and `OCI_WORKER_PV_TRANSIT_ENCRYPTION` parameters disable encryption of data in flight between the bare metal instance and the block storage resources.
```

The user experience with in-flight encryption is somewhat hostile: if set to true and the instance shape doesn't support it, we should just fail. That provides the best security and fast-fail result.

Reply:
> if set to true and the instance shape doesn't support it, we should just fail.

https://github.com/oracle/cluster-api-provider-oci/issues/34